### PR TITLE
Init

### PR DESCRIPTION
**Pull request by michz**

### CODE REVIEW

File: src/Command/DownloadCommand.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DownloadCommand extends Command
+{
+    /** @var S3ClientFactory */
+    private $clientFactory
+    ;
```
Comment (jakoberr) [inline src/Command/DownloadCommand.php]: Is this on purpose?

**Review by jakoberr — DISMISSED** [review]

File: src/Command/DownloadCommand.php (hunk 1 above)
Comment (michz) [reply src/Command/DownloadCommand.php]: There are people out there assuming I steal newlines and waste them...

Comment (jakoberr) [reply src/Command/DownloadCommand.php]: How dare they!

**Review by Jnoack331 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+
+
+Author
+------
+
+solutionDrive GmbH
+info@solutionDrive.de
+
+
+License
```
Comment (sd-lueckel) [inline README.md:51]: I would like it if you add a 'DEVELOP' section like here https://github.com/solutionDrive/sdSwPluginManager/blob/master/README.md

**Review by sd-lueckel — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (michz) [reply README.md:51]: Done.

**Review by sd-lueckel — APPROVED** [review]

**Pull request merged**